test/serialization.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('serialization', () => {
     expect(engine1.getCellValueDetailedType(adr('F1'))).toEqual(CellValueDetailedType.NUMBER_CURRENCY)
 
     expect(engine1.getCellSerialized(adr('G1'))).toEqual('12/01/15')
-    expect(engine1.getCellValueFormat(adr('G1'))).toEqual(undefined)  //this is wrong, see issue 626
+    expect(engine1.getCellValueFormat(adr('G1'))).toEqual('DD/MM/YY')
```

Comment:
I've seen that you handled native `Date()` object. Will it work with serialization? Can we add an example to the dataset in this test?